components/appmgt/org.wso2.carbon.appmgt.impl/src/main/java/org/wso2/carbon/appmgt/impl/utils/RESTAPIAdminClient.java

```diff
@@ -102,21 +108,112 @@ public void updateApi(APITemplateBuilder builder, String tenantDomain) throws Ax
 	}
 
 	/**
-	 * Delete the WebApp from Gateway
+	 * Deletes versioned web app configuration from the gateway
+	 *
 	 * @param tenantDomain
 	 * @throws AxisFault
 	 */
-	public void deleteApi(String tenantDomain) throws AxisFault {
+	public void deleteVersionedWebApp(String tenantDomain) throws AxisFault {
 		try {
 			if (tenantDomain != null && !("").equals(tenantDomain) &&
-			    !tenantDomain.equals(MultitenantConstants.SUPER_TENANT_DOMAIN_NAME)) {
+					!tenantDomain.equals(MultitenantConstants.SUPER_TENANT_DOMAIN_NAME)) {
```

Comment:
@imesh I believe you meant to comment here. 
Looks like this happens due to tabs which were used to indent the code. However this line has been changed when correcting other issues in this PR. See commit 50d5da1ffa1fffacf500b8005728d72302726392 of PR #153